pipeline/splicing/test_calcVarPriors.py

```diff
@@ -94,6 +95,10 @@
                 "acceptors": {"std": 2.4336623152078452,
                               "mean": 7.984909090909091}}
 
+# standard exonic portion size and de novo length
+stdExonicPortion = 3
```

Comment:
you should import these directly from calcVarPriors.py, e.g. `from calcVarPriors import stdExonicPortion, stdDeNovoLength`